hawkular-nest/hawkular-nest-itest-parent/hawkular-nest-itest/pom.xml

```diff
@@ -0,0 +1,170 @@
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.hawkular.commons</groupId>
+    <artifactId>hawkular-nest-itest-parent</artifactId>
+    <version>0.2.5.Final-SNAPSHOT</version>
+  </parent>
+  <artifactId>hawkular-nest-itest</artifactId>
+  <name>Hawkular Nest Intagration Tests</name>
+
+  <properties>
+    <hawkular.nest.deployments.dir>${project.build.directory}/${project.artifactId}-${project.version}/modules/org/hawkular/nest/main/deployments</hawkular.nest.deployments.dir>
+    <hawkular.nest.itest.war.path>${hawkular.nest.deployments.dir}/hawkular-nest-itest-war-${project.version}.war</hawkular.nest.itest.war.path>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jboss.arquillian</groupId>
+        <artifactId>arquillian-bom</artifactId>
+        <version>1.1.10.Final</version>
+        <scope>import</scope>
+        <type>pom</type>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>com.squareup.okhttp</groupId>
+      <artifactId>okhttp</artifactId>
+      <scope>test</scope>
```

Comment:
Very small thing: I personally prefer to keep the "test" dependencies together, as it gives me a better idea on what's needed for the application and what's needed for the test.